store/src/java/com/zimbra/util/ExternalVolumeInfoHandler.java

```diff
@@ -360,4 +376,18 @@ public Boolean isVolumePresentInJson(int volumeId) throws ServiceException, JSON
         }
         return false;
     }
+
+    /**
+     * Performs flush cache at config level on all the server
+     * @param adminDocumentHandler, context
+     * @return nothing
+     */
+    public static void flushConfigLevelCacheOnAllServers(AdminDocumentHandler adminDocumentHandler, Map<String, Object> context) {
+        CacheSelector cacheSelector = new CacheSelector(true, CacheEntryType.config.toString());
+        try {
+            FlushCache.doFlushCache(adminDocumentHandler, context, new FlushCacheRequest(cacheSelector));
+        } catch (ServiceException se) {
+            ZimbraLog.account.error("Encountered exception during FlushCache: ", se);
```

Comment:
not sure we should log to account category as this operation is not related to any account

Reply:
Done. Changed it to `misc` as used in FlushCache utility.